src/main/java/com/redhat/coolstore/rest/RestApplication.java

```diff
@@ -1,10 +1,5 @@
 package com.redhat.coolstore.rest;
 
-import javax.ws.rs.ApplicationPath;
-import javax.ws.rs.core.Application;
+public class RestApplication {
 
-
-@ApplicationPath("/services")
```

Comment:
This is surprisingly right according to [our rules](https://github.com/windup/windup-rulesets/blob/master/rules/rules-reviewed/quarkus/java-ee/jaxrs-to-quarkus.windup.xml#L46), but it's not present in Chris' migration. What do you think @christophermay07 ? I am not sure if the fact that a path is specified (`/services`) is important here.

The rule we have is `optional`, so it is something that _can be done_ but isn't necessary. This could potentially be something to tell the LLM too @jwmatthews